src/main/java/org/example/MainServlet.java

```diff
@@ -20,8 +23,8 @@ public class MainServlet extends HttpServlet {
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-        logRequest(req);
 
+        LOGGER.info("We enter on path {}", req);
```

Comment:
Does it work? Could you please provide the output? Is the output informative enough?

Reply:
Fixed